spec/amp-var-substitutions.md

```diff
@@ -65,6 +65,37 @@ For instance:
 ```
 may make a request to something like `https://foo.com/pixel?path=%2Fpage1.html`.
 
+**CLIENT_ID**
+
+Use the special string `CLIENT_ID` to add a per document-source-origin (The origin of the website where you publish your AMP doc) and user identifier. The CLIENT_ID will be the same for the same user if they visit again within one year. It should behave roughly similar to a cookie storing a session id for one year. Please take note of the allowed arguments you may
+pass into `CLIENT_ID` like a function. (spaces between arguments and values are not allowed)
+
```

Comment:
One more sentence:

If the AMP document is not served through the AMP CDN, the CLIENT_ID will be replaced with a cookie of the name of the `cid scope` below. If the cookie is not present, the empty string will be returned.